pkg/reconciler/autoscaling/kpa/kpa_test.go

```diff
@@ -186,6 +186,10 @@ func markUnknown(pa *asv1a1.PodAutoscaler) {
 	pa.Status.MarkActivating("", "")
 }
 
+func markReconcile(pa *asv1a1.PodAutoscaler) {
```

Comment:
As this is used in the other test as well, maybe this should be added to one of the common files?

Reply:
You might also want to consider to set this as the default when calling `kpa`. That'd keep the diff way more minimalistic and gets out of the way for future changes.